modules/gdnative/nativescript/nativescript.cpp

```diff
@@ -1905,6 +1905,9 @@ void NativeReloadNode::_notification(int p_what) {
 				}
 
 				for (Map<String, Set<NativeScript *>>::Element *U = NSL->library_script_users.front(); U; U = U->next()) {
+					if (L->key() != U->key())
```

Comment:
Maybe there should be a comment above this `if` statement? It's not 100% obvious why it's needed.